src/app/app.component.html

```diff
@@ -22,37 +22,38 @@
       <md-accordion displayMode="flat" [multi]="'multi'" class="md-expansion-menu">
         <md-expansion-panel #panel1 expanded="true">
           <md-expansion-panel-header>Wallet</md-expansion-panel-header>
-
           <md-content layout-padding class="submenuarea">
             <md-list class="sidesubmenu">
-              <a routerLinkActive="active" [routerLink]="walletInitialized ? '/wallet/receive' : null">
+              <a routerLinkActive="active" routerLink="/wallet/receive">
+              <!-- Todo: fix walletInitialized in combination with active -->
```

Comment:
This might do the trick
`routerLinkActive="{{ walletInitialized ? 'active' : null }}" [routerLink]="walletInitialized ? '/wallet/receive' : null"
`